addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -1595,6 +1595,8 @@
                     <t t-esc="receipt.footer" />
                 </div>
             </t>
+
+            <div class='after-footer' />
```

Comment:
maybe not necessary, select . pos-sale-ticket and append inside, it should go at the end